infra/gcb/templates/src/build-status/build-status.html

```diff
@@ -191,23 +272,48 @@
       onTap(e) {
         // Change the route, this should auto-magically update the url in the
         // browser and invoke the _routeChanged method.
-        this.set('route.path', e.model.project.name);
+        this.set("route.path", e.model.project.name);
       }
 
       onChanged(e) {
-        if (this.$.build_type.selected == 'coverage') {
-          this.status = this.status_coverage
+        if (this.$.build_type.selected == "coverage") {
+          this.status = this.status_coverage;
         } else {
-          this.status = this.status_fuzzing
+          this.status = this.status_fuzzing;
         }
       }
 
-      showMessage(loading_log, log) {
-        return !loading_log && log === '';
+      onBuildHistoryClick(e) {
+        var ajax = this.$.logxhr;
+        const id = e.model.__data.history.build_id;
```

Comment:
accessing __data seems wrong as the __ indicates it's private.

Per https://polymer-library.polymer-project.org/2.0/docs/devguide/templates#handling-events, can you access this via e.model.item instead?